### ISSUE TITLE
can't start endlessh service "Failed at step NAMESPACE"

### ISSUE THREAD

**Issue by KNG6** [skeeto/endlessh#66]
journalctl logs:

```
May 02 21:48:28 [REACTED] systemd[1]: Started Endlessh SSH Tarpit.
May 02 21:48:29 [REACTED] endlessh[16317]: endlessh.service: Failed at step NAMESPACE spawning /usr/local/bin/endlessh: No such file or directory
May 02 21:48:29 [REACTED] systemd[1]: endlessh.service: Main process exited, code=exited, status=226/NAMESPACE
May 02 21:48:29 [REACTED] systemd[1]: endlessh.service: Unit entered failed state.
May 02 21:48:29 [REACTED] systemd[1]: endlessh.service: Failed with result 'exit-code'.
```

right of the file:
`-rwxr-xr-x 1 root root 128800 May  2 20:21 /usr/local/bin/endlessh`

endlessh.service:
```
[Unit]
Description=Endlessh SSH Tarpit
Documentation=man:endlessh(1)
Requires=network-online.target

[Service]
Type=simple
Restart=always
RestartSec=30sec
ExecStart=/usr/local/bin/endlessh
KillSignal=SIGTERM

# Stop trying to restart the service if it restarts too many times in a row
StartLimitInterval=5min
StartLimitBurst=4

StandardOutput=journal
StandardError=journal
StandardInput=null

PrivateTmp=true
PrivateDevices=true
ProtectSystem=full
ProtectHome=true
InaccessiblePaths=/run /var

## If you want Endlessh to bind on ports < 1024
## 1) run: 
##     setcap 'cap_net_bind_service=+ep' /usr/local/bin/endlessh
## 2) uncomment following line
AmbientCapabilities=CAP_NET_BIND_SERVICE
## 3) comment following line
#PrivateUsers=true

NoNewPrivileges=true
ConfigurationDirectory=endlessh
ProtectKernelTunables=true
ProtectKernelModules=true
ProtectControlGroups=true
MemoryDenyWriteExecute=true

[Install]
WantedBy=multi-user.target
```
With the command, it's work perfectly

If you have some idea ?

**Comment by DigitalDragon64** [comment]
Seems that the systemd modul is misconfigured at the moment, similar to
[https://github.com/skeeto/endlessh/issues/53](https://github.com/skeeto/endlessh/issues/53)

comment the line
`#InaccessiblePaths=/run /var`
did solve it on my machine.

**Comment by KNG6** [comment]
> Seems that the systemd modul is misconfigured at the moment, similar to
> #53
> 
> comment the line
> `#InaccessiblePaths=/run /var`
> did solve it on my machine.

Thanks that solve my issue !

**Comment by nidamanx** [comment]
> Seems that the systemd modul is misconfigured at the moment, similar to #53
> 
> comment the line `#InaccessiblePaths=/run /var` did solve it on my machine.

Issue is still there on Debian 12
I'm wondering if we could brake some security while that line is commented
Maybe @skeeto could give us more precise infos
Thanks!

**Comment by chjohans** [comment]
I came across this thread as I just upgraded from Ubuntu 22.04 to 24.04 LTS on my main server, and it has been running endlessh for a few years now. But after the upgrade it failed to start as a service with the same error as described in this issue.

After commenting out "InaccessiblePaths=/run /var" in endlessh.service is starts and seems to work just fine, but like other in this thread I am a bit concerned about possible security implications from making that change.

Care to comment on this @skeeto?
